slu-scripts/src/device-booting/runSendData.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from 'fs';
+import { execSync } from 'child_process';
+import 'dotenv/config';
+
+const keyFile = process.env.KEY_FILE;
+const inputEnc = process.env.INPUT_ENC;
+const isApiKey = process.env.IS_API_KEY;
+const isBaseUrl = process.env.IS_BASE_URL;
+const isAuthUrl = process.env.IS_AUTH_URL;
+const sendDataInterval = process.env.SEND_DATA_INTERVAL;
+const sensorData = process.env.SENSOR_DATA;
+const collectorBaseUrl = process.env.COLLECTOR_BASE_URL;
+const scriptsPath = process.env.SCRIPTS_PATH;
+
+export function runSendData(
+	keyFile: string | undefined,
+	inputEnc: string | undefined,
+	isApiKey: string | undefined,
+	isBaseUrl: string | undefined,
+	isAuthUrl: string | undefined,
+	sendDataInterval: string | undefined,
+	sensorData: string | undefined,
+	collectorBaseUrl: string | undefined,
+	scriptsPath: string | undefined
```

Comment:
this wont be checked in the if, was this forgotten or is it ok to be '' ?

Reply:
yes, it must be included